libraries/GCS_MAVLink/GCS.cpp

```diff
@@ -37,7 +37,7 @@ const MAV_MISSION_TYPE GCS_MAVLINK::supported_mission_types[] = {
  */
 void GCS::send_textv(MAV_SEVERITY severity, const char *fmt, va_list arg_list)
 {
-    char text[MAVLINK_MSG_STATUSTEXT_FIELD_TEXT_LEN+1];
+    char text[256+1]; // eep!  Stack!  Eeep!
```

Comment:
as send_text() can be called from any thread, I think this is too much stack to use

Reply:
Fixed.  We now use a buffer shared buffer.